fern/docs/pages/integrations/stripe-integration-guide.mdx

```diff
@@ -0,0 +1,128 @@
+4. (Less common) If you have an identifier set in your companies's Stripe metadata (often your application database's id), Schematic can import this and use it to sync companies during setup. 
+
+<Info>
+To understand more about `identify` and keys, see the [Developer Resources - Keys](/developer_resources/key) page.
+</Info>
+
+#### Using a Company identifier from Stripe metadata
+
+If you have have a company identifier already stored in Stripe metadata, Schematic can import that along with the companies. After connecting Stripe, choose the 2nd option so that Schematic can pull the correct key from Stripe. 
+
+![Stripe Customer Key from Metadata](../../assets/images/stripe/stripe-customer-key-metadata-1.png)
+
+Then, select which metadata value you want Schematic to pull.
+
+![Stripe Customer Key from Metadata 2](../../assets/images/stripe/stripe-customer-key-metadata-2.png)
+
+When importing companies this way, a few things to note:
+1. Schematic will check existing companies for a key with the same name and value as the metadata field you selected. On a match, it will update the existing company instead of creating a new one.
+2. When a new company is created, it will set both `stripe_customer_id` and the metadata key you selected.
+3. Going forward, Schematic will still priotizie `stripe_customer_id` as the key it uses to sync companies with Stripe.
+
+### Prod vs. Dev Stripe Accounts
+
+Schematic supports Stripe accounts, sandboxes, and legacy test accounts. Each account is scoped to a specific Schematic Environment (e.g. you can connect a normal Stripe account to your production Schematic environment, a Sandbox to your development Schematic environment, and a different Sandbox to your QA environment). We don't recommend connecting the same Sandbox to multiple Schematic environments.
+
+**Prod Stripe Account**
+
+This the normal Stripe account you use for your production environment.
+
+![Stripe Prod Account](../../assets/images/stripe/stripe-prod-setup.png)
+
+**Sandbox Stripe Account**
+
+![Stripe Sandbox Account](../../assets/images/stripe/stripe-sandbox-setup.png)
+
+**Legacy Test Account**
+
+Stripe has deprecated these legacy test accounts in favor of the new Sandbox system.
+
+![Stripe Legacy Test Account](../../assets/images/stripe/stripe-legacy-test-setup.png)
+
+### What existing after setup
+
+After setting up the integration, the Stripe Integration page in Schematic will show you the following:
+
+- A list of imported Products
+- A list of imported Companies
+- A button to remove the connection. No data will be deleted from either system.
+- A button to refresh the data. This will re-import all products and companies from Stripe.
+
+<Info>Schematic should receive updates from Stripe automatically, in real-time, via webhooks. The "Refresh data" button should only be used for troubleshooting.</Info>
+
+![Stripe Imported Products](../../assets/images/stripe/stripe-import-view.png)
+
+### Stripe Restricted Key permissions
```

Comment:
Can we remove this? Our users are no longer issuing keys to send to us, they always use Stripe connect, so I think this information is no longer relevant